### PR TITLE
[DOC] Edit monitoring

### PR DESCRIPTION
**Pull request by MagdaDziadosz**
A slightly bigger edit, added descriptions and tables.

### CODE REVIEW

**Review by nsteph — APPROVED** [review]
LGTM. Thanks!

File: content/docs/reference/monitoring/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Monitoring
+
+This document provides guidance on setting up monitoring for your
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:3]: Suggest losing this intro-ish sentence. as the heading does the same job, given the topic context.

File: content/docs/reference/monitoring/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Kubernetes clusters using Mirantis Kubernetes Engine (MKE) 4. 
+The monitoring setup is based on the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack),
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:5]: Tap 'MKE 4' in front of 'monitoring'.

File: content/docs/reference/monitoring/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+offering a comprehensive solution for collecting, storing, and visualizing metrics.
+
+## Monitoring tools
+
+Descriptions for each of the monitoring tools that MKE 4 uses are
+available in the table below.
+
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:12]:
```suggestion
Detail for the MKE 4 monitor tools is provided in the following table:

```

File: content/docs/reference/monitoring/monitoring.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+| Monitoring tool    | Default State | Configuration Key          | Description                                                                    |
+|------------|---------------|----------------------------|--------------------------------------------------------------------------------|
+| Grafana    | enabled       | `monitoring.enableGrafana` | Provides a web interface for viewing metrics and logs collected by Prometheus  |
+| Prometheus | enabled       | -                          | Collects and stores metrics                                                    |
+| Opscare    | disabled      | `monitoring.enableOpscare` | (Future feature) Includes additional monitoring capabilities like Alertmanager |
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:17]:
```suggestion
| Opscare    | disabled      | `monitoring.enableOpscare` | (Future feature) Supplies additional monitoring capabilities, such as Alertmanager |
```

File: content/docs/reference/monitoring/monitoring.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+### Grafana
+
+[Grafana](https://grafana.com/) is an open-source platform for monitoring.
+It provides a rich set of tools for visualizing time-series data,
+including a variety of graph types and dashboards.
+
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:24]:
```suggestion
[Grafana](https://grafana.com/) is an open-source monitoring platform that provides a rich set of tools for visualizing time-series data. It
includes a variety of graph types and dashboards.

```

File: content/docs/reference/monitoring/monitoring.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Grafana is enabled in MKE 4 by default and may be disabled through the configuration:
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:25]:
```suggestion
Grafana is enabled in MKE 4 by default and may be disabled through the MKE configuration file:
```

File: content/docs/reference/monitoring/monitoring.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+To access Grafana dashboard:
+
+1. Run the following command to `port-forward` Grafana:
+
+    ```bash
+    kubectl --namespace mke port-forward svc/monitoring-grafana 3000:80
+    ```
+
+2. Access Grafana dashboard through [http://localhost:3000](http://localhost:3000).
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:40]:
```suggestion
2. Go to [http://localhost:3000](http://localhost:3000).
```

File: content/docs/reference/monitoring/monitoring.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+### Prometheus
+
+[Prometheus](https://prometheus.io/) is an open-source monitoring and alerting
+toolkit designed for reliability and scalability. It collects and stores metrics
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:45]:
```suggestion
toolkit that is designed for reliability and scalability. It collects and stores metrics
```

File: content/docs/reference/monitoring/monitoring.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+including a variety of graph types and dashboards.
+
+Grafana is enabled in MKE 4 by default and may be disabled through the configuration:
+
+```yaml
+monitoring:
+  enableGrafana: true
+```
+
+To access Grafana dashboard:
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:32]:
```suggestion
To access the Grafana dashboard:
```

File: content/docs/reference/monitoring/monitoring.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+2. Access Grafana dashboard through [http://localhost:3000](http://localhost:3000).
+
+### Prometheus
+
+[Prometheus](https://prometheus.io/) is an open-source monitoring and alerting
+toolkit designed for reliability and scalability. It collects and stores metrics
+as time series data, providing powerful query capabilities and a flexible alerting system.
+
+To access Prometheus dashboard:
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:48]:
```suggestion
To access the Prometheus dashboard:
```

File: content/docs/reference/monitoring/monitoring.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Run the following command to `port-forward` Prometheus:
+
+    ```bash
+    kubectl --namespace mke port-forward svc/prometheus-operated 9090
+    ```
+
+2. Access Prometheus dashboard through [http://localhost:9090](http://localhost:9090).
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:56]:
```suggestion
2. Go to [http://localhost:9090](http://localhost:9090).
```

File: content/docs/reference/monitoring/monitoring.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+## Opscare (Feature in progress)
+
+[Mirantis Opscare](https://www.mirantis.com/resources/opscare-datasheet/) is
+an advanced monitoring and alerting solution. It enhances the monitoring
+capabilities of MKE 4 by integrating additional tools and features, such as
+[Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/).
+Future releases will include more integrations, such as a Salesforce-notifier component.
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:64]: Suggest removing this line. Need to refrain from making promises for the future.

File: content/docs/reference/monitoring/monitoring.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+    ```
+
+2. Access Prometheus dashboard through [http://localhost:9090](http://localhost:9090).
+
+## Opscare (Feature in progress)
+
+[Mirantis Opscare](https://www.mirantis.com/resources/opscare-datasheet/) is
+an advanced monitoring and alerting solution. It enhances the monitoring
+capabilities of MKE 4 by integrating additional tools and features, such as
+[Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/).
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:63]:
```suggestion
an advanced monitoring and alerting solution. Once it is integrated, Mirantis Opscare will enhance the monitoring
capabilities of MKE 4 by incorporating additional tools and features, such as
[Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/).
```

File: content/docs/reference/monitoring/monitoring.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+Future releases will include more integrations, such as a Salesforce-notifier component.
+
+Mirantis OpsCare is disabled by default and may be enabled through the MKE4 configuration.
```
Comment (KoryKessel-Mirantis) [inline content/docs/reference/monitoring/monitoring.md:66]: Disabled by default, you will enable Mirantis Opscare through the MKE configuration file.
```suggestion
Disabled by default, you will enable Mirantis Opscare through the MKE configuration file.
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

File: content/docs/reference/monitoring/monitoring.md (hunk 14 above)
Comment (MagdaDziadosz) [reply content/docs/reference/monitoring/monitoring.md:66]:
```suggestion
Disabled by default, you can enable Mirantis Opscare through the MKE configuration file.
```

**Comment by KoryKessel-Mirantis** [comment]
LGTM

**Review by KoryKessel-Mirantis — APPROVED** [review]
LGTM

**Pull request merged**